package.json

```diff
@@ -29,7 +29,7 @@
     "Blame",
     "Git"
   ],
-  "icon": "images/logo.png",
```

Comment:
Is "logo" being used anywhere else?
why not replace the file?

Reply:
It's not used elsewhere. 

<img width="361" alt="Screen Shot 2022-08-25 at 12 25 42 PM" src="https://user-images.githubusercontent.com/8325094/186751225-1533a6ff-7c06-4538-bf8d-c193b5164955.png">